includes/view/User_view.php

```diff
@@ -83,13 +84,53 @@ function Users_view(
     $force_active_count,
     $freeloads_count,
     $goodies_count,
-    $voucher_count
+    $voucher_count,
+    $admin_user_privilege
 ) {
     $goodie = GoodieType::from(config('goodie_type'));
     $goodie_enabled = $goodie !== GoodieType::None;
     $goodie_tshirt = $goodie === GoodieType::Tshirt;
     $usersList = [];
     foreach ($users as $user) {
+        $voucher_field = '';
+
+        if (config('enable_voucher')) {
+            $voucher_template = <<<EOT
+            <div class="d-flex align-items-center text-nowrap js-only">
+                <div data-field="voucher-status" class="flex-grow-1 text-end">
+                    {issued} / {eligible}
+                </div>
+                {plus_one}
+            </div>
+EOT;
+
+            if (($admin_user_privilege || $auth->can('voucher.edit')) && config('enable_voucher')) {
```

Comment:
`config('enable_voucher')` is redundant at that point & `$auth` not defined

Reply:
Done